app/assets/javascripts/discourse/app/components/user-card-contents.js

```diff
@@ -18,11 +18,11 @@ import { escapeExpression } from "discourse/lib/utilities";
 import CanCheckEmails from "discourse/mixins/can-check-emails";
 import CleansUp from "discourse/mixins/cleans-up";
 import User from "discourse/models/user";
-import { getURLWithCDN } from "discourse-common/lib/get-url";
+import getURL, { getURLWithCDN } from "discourse-common/lib/get-url";
 import discourseComputed from "discourse-common/utils/decorators";
 import I18n from "discourse-i18n";
 
-@classNames("user-card")
+@classNames("d-user-card")
```

Comment:
I had a quick look at all-the-themes/all-the-plugins, and I get results from 62 files for a search for `.user-card`. Meaning, that a lot of themes and plugins reference the old class. Sometimes it's the parent, and other times it's `.user-card-avatar`, `.user-card-button` etc. 

It may make sense to keep the old class name, at least as a first step. As is, this risks breaking too many customizations.

Reply:
I initially wanted to keep the original only, but having a clean start helps me kick off things better. I'll add both for now :+1: